packages/swapper/src/swappers/cow/utils/helpers/helpers.ts

```diff
@@ -75,11 +75,7 @@ export const getUsdRate = async (
 }
 
 export const getNowPlusThirtyMinutesTimestamp = (): number => {
-  const millisecondsPerMinute = 60000
-
-  // UTC date + 30 minutes
-  const nowPlusThirtyMinutesDate = new Date(new Date().getTime() + 30 * millisecondsPerMinute)
-
-  // returning the timestamp in seconds
-  return Math.round(nowPlusThirtyMinutesDate.getTime() / 1000)
+  const ts = new Date()
+  ts.setMinutes(ts.getMinutes() + 30)
```

Comment:
> this will still break twice a day every four years but i'm ok with that 😛

Found those DST issues as well but it seems very old ? maybe it is not an issue anymore ?